core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -64,7 +64,7 @@ object ZkUtils {
   val BrokerSequenceIdPath = s"$BrokersPath/seqid"
   val ConfigChangesPath = s"$ConfigPath/changes"
   val ConfigUsersPath = s"$ConfigPath/users"
-  val PidBlockPath = "/latest_pid_block"
+  val ProducerIdBlockPath = "/latest_pid_block"
```

Comment:
Here's a PR that does this and a few other renames: https://github.com/apache/kafka/pull/3041